spec/views/documents/_timeline_entry.html.erb_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+RSpec.describe "documents/_timeline_entry.html.erb" do
+  describe "timeline entry" do
```

Comment:
I don't think this section needs this describe as all things are timeline entries. I'd skip having a describe here and just have its within the main describe.